Compile warning on VS2022

I am getting this compiler warning on VS2022 about not enough bits in a bitfield interfacing with glslang b0df68c490363ac347e454f79886293f661ce4f6 from our C++ code:

```
1>H:\dev\source2\main\src\thirdparty\glslang\glslang\Include\Types.h(576,25): error C2220: the following warning is treated as an error
1>(compiling source file 'vulkan/renderdevicevulkan.cpp')
1>H:\dev\source2\main\src\thirdparty\glslang\glslang\Include\Types.h(576,25): warning C5249: 'glslang::TQualifier::storage' of type 'glslang::TStorageQualifier' has named enumerators with values that cannot be represented in the given bit field width of '6'.
1>(compiling source file 'vulkan/renderdevicevulkan.cpp')
1>H:\dev\source2\main\src\thirdparty\glslang\glslang\Include\Types.h(576,25):
1>to simplify migration, consider the temporary use of /Wv:18 flag with the version of the compiler with which you used to build without warnings
1>H:\dev\source2\main\src\thirdparty\glslang\glslang\Include\BaseTypes.h(137,5):
1>see enumerator 'glslang::EvqLast' with value '32'
```

--- a/glslang/Include/Types.h
+++ b/glslang/Include/Types.h
@@ -573,7 +573,8 @@ public:
     }
 
     const char*         semanticName;
-    TStorageQualifier   storage   : 6;
+    TStorageQualifier   storage   : 7;
+    static_assert(EvqLast < 64, "need to increase size of TStorageQualifier bitfields!");
     TBuiltInVariable    builtIn   : 9;
     TBuiltInVariable    declaredBuiltIn : 9;
     static_assert(EbvLast < 256, "need to increase size of TBuiltInVariable bitfields!");

